app/controllers/actions/IdentifierAction.scala

```diff
@@ -150,61 +152,44 @@ class AuthenticatedIdentifierAction @Inject() (
         Future.successful(Redirect(config.viewAndChangeEnterUtrUrl))
     }
 
-  private def agentRecovery[A](
-    block: IdentifierRequest[A] => Future[Result],
-    internalId: String,
-    mtdItId: String,
-    nino: String
-  )(implicit request: Request[A], hc: HeaderCarrier): PartialFunction[Throwable, Future[Result]] = {
-    case _: AuthorisationException if config.emaSupportingAgentsEnabled =>
+  private def agentRecovery[A](block: IdentifierRequest[A] => Future[Result], internalId: String, mtdItId: String, nino: String)(implicit
+                                                                                                                                 request: Request[A],hc: HeaderCarrier): PartialFunction[Throwable, Future[Result]] = {
+    case _: AuthorisationException =>
       authorised(secondaryAgentPredicate(mtdItId))
         .retrieve(allEnrolments) {
           populateAgent(block, internalId, mtdItId, nino, _, isSupportingAgent = true)
         }
         .recoverWith {
           case _: AuthorisationException =>
-            logger.warn(
-              s"[AuthorisedAction][agentAuthentication] - Agent does not have secondary delegated authority for Client."
-            )
+            logger.warn(s"$agentAuthLogString - Agent does not have secondary delegated authority for Client.")
```

Comment:
this message doesn't look right to me. it can be simple agent not authorised? as we are not supporting secondary agent now